pom.xml

```diff
@@ -303,7 +291,7 @@
         <dependency>
             <groupId>org.eclipse.uprotocol</groupId>
             <artifactId>uprotocol-core-api</artifactId>
-            <version>1.5.1</version>
+            <version>1.5.2-SNAPSHOT</version>
```

Comment:
why did you bump the version to a SNAPSHOT?

Reply:
It needs to be snapshot and then the release script will create versions without the SNAPSHOT tag